health/openapi.json

```diff
@@ -0,0 +1,89 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Health API",
+    "description": "Health HTTP endpoints: readyz, healthz, version on their default paths (see [docs](https://pkg.go.dev/github.com/anz-bank/pkg/health#example-package)).",
+    "version": "0.0.1"
+  },
+  "paths": {
+    "/healthz": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "text/plain": {
+                "schema": {
+                  "type": "string"
+                },
+                "example": "200 ok"
+              }
+            }
+          },
+          "503": {
```

Comment:
this belongs on `/readyz`. `/healthz` always returns 200 if it returns anything

Reply:
sorry. i blame fabric.